greedy_approach/JobSequencing.c

```diff
@@ -0,0 +1,79 @@
+#include<stdio.h>
+
+struct Jobs
+{
+    float profit;
+    int deadline;
+    int id;
```

Comment:
Please add a one-line description of what the variable is for (**see the example below**).

```c
int64_t data = 0;       ///< The value/key of the node.
int64_t array[64];      ///< The array that will be used for the tree.
```